.github/workflows/tests.yml

```diff
@@ -270,6 +270,109 @@ jobs:
             test-report.xml
           retention-days: 1  # temporary, combined in aggregate below
 
+  # linux-arm64 test suite
+  linux-arm64:
```

Comment:
is this a copy past from the `linux` job with minimal adjustments? if so I think we should just amend the `linux` job and add `ubuntu-24.04-arm` to the test matrix?

Reply:
It can be: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#context-availability

<img width="742" alt="Screenshot 2025-01-31 at 15 12 35" src="https://github.com/user-attachments/assets/266276f4-6ee6-4463-8ade-019afbee59ce" />